### PR TITLE
Mark unload as deprecated

### PR DESCRIPTION
**Pull request by tunetheweb**
#### Summary

The `unload` event is unreliable on mobile (all browsers) and desktop (safari) [where it only fires ~60% of the time](https://github.com/fergald/docs/blob/master/explainers/permissions-policy-deprecate-unload.md#unload-reliability).

Chrome has stated it's intent to deprecate it: https://developer.chrome.com/docs/web-platform/deprecating-unload and it was previously proposed to mark it as not supported in https://github.com/mdn/browser-compat-data/pull/20597 - which was correctly closed as too early, and instead a documentation issue was raised to discuss this: https://github.com/mdn/mdn/issues/462.

This PR proposes leaving the support as is (with a note for Chrome), but marking it as deprecated to warn developers more that it should not be relied on.

It should be noted that Safari and Firefox are not quite on board with the path to get there:
- Safari position: https://github.com/WebKit/standards-positions/issues/200 but [seems they agree in principal to deprecation](https://github.com/WebKit/standards-positions/issues/200#issuecomment-1591547018)
- Firefox position: https://github.com/mozilla/standards-positions/issues/691 

However, the MDN documentation page has had a warning at the top of it's page for as long as that repo existed (3 years):
https://developer.mozilla.org/en-US/docs/Web/API/Window/unload_event

image

So, given it is unrealiable on mobile for all browsers and desktop Safari, that Chrome intends to deprecate it fully, and that we have had a warning at the top of the documentation page for a long time, it seems to me [it fits the MDN definition of deprecated](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Experimental_deprecated_obsolete#deprecated):

> The term **deprecated** on MDN Web Docs is used to mark an API or technology that is no longer recommended. A deprecated API or technology might be removed in the future or might only be kept for compatibility purposes and may still work. We recommend to avoid using the functionality marked as deprecated.

It has not yet been formally deprecated in the HMTL spec, which might be more problematic for [the BCD definition of deprecated](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#setting-deprecated) which prefers an actual specification update or at least a position.

Discussions in the spec have been ongoing for a while:
https://github.com/whatwg/html/issues/6026

The spec has clarified that the unload handler [does not need to run any more when the bfcache can be used](https://github.com/whatwg/html/pull/5889) - in effect codifying the mobile and Safari unreliability.

TLDR: it's messy but this event really is not recommended to be used (as shown by the warning on MDN already) and that is only likely to increase.

#### Test results and supporting details

See above

#### Related issues

Issue to document this deprecation journey: https://github.com/mdn/mdn/issues/462

### CODE REVIEW

File: api/Window.json [hunk 1]
```diff
@@ -6432,7 +6432,8 @@
           ],
           "support": {
             "chrome": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "Starting in Chrome 117, the [unload event is on the path to deprecation](https://developer.chrome.com/docs/web-platform/deprecating-unload)."
```
Comment (queengooborg) [inline api/Window.json:6436]: We don't typically mention when a feature is deprecated in a web browser, only when its support has been removed.  I think we can remove this note.

Comment (tunetheweb) [reply api/Window.json]: Updated.

**Review by queengooborg — APPROVED** [review]
Thanks, LGTM!
